network-api/networkapi/wagtailpages/factory/buyersguide.py

```diff
@@ -179,7 +179,7 @@ def set_random_review_date(self, create, extracted, **kwargs):
     @post_generation
     def set_random_creepiness(self, create, extracted, **kwargs):
         self.get_or_create_votes()
-        single_vote = [0, 0, 0, 0, 0]
+        single_vote = [0, 0, 1, 0, 0]
         shuffle(single_vote)
         self.votes.set_votes(single_vote)
         self.creepiness_value = randint(0, 100)
```

Comment:
If we do not have one vote the avg product creepiness is the same, which makes the sort useless without manually voting.

Reply:
Didn't we previously already sort by creepiness?

Reply:
And isn't the creepiness set in the last row to a random int?

Reply:
The row doesn't matter since it gets shuffled anyways. If you use the factories that are already on pni branch and Inv new db and inspect the products you will notice all of them having the same creepiness value without having this fix. I think that will help make sense of this change

Reply:
I can see that the shuffle should be useless previously, since it's all the same value... So you still get the same list after. What I don't understand is what that has to do with the creepiness value. Is the creepiness value not set by the last line?

Reply:
I don't understand what is happening here. All the naming if completely misleading: https://github.com/mozilla/foundation.mozilla.org/blob/866eeef72d8357fcbeb4d68ebe87be34243eda1a/network-api/networkapi/wagtailpages/pagemodels/buyersguide/products.py#L746-L760